annotators/user_knowledge_memorizer/server.py

```diff
@@ -451,12 +470,81 @@ def check_and_add_properties(graph, prop_triplets: List[dict], user_id: str) ->
     return properties_to_add_to_kg, properties_already_in_kg
 
 
+def get_knowledge(user_id):
+    user_data = kg_graph.search_for_relationships(id_a=user_id)
+    rels = [triplet["rel"] for triplet in user_data]
+    entity_ids = [triplet["id_b"] for triplet in user_data]
+    relationship_entity_id_pair = list(zip(rels, entity_ids))
+    entity_values = kg_graph.get_properties_of_entities(entity_ids)
+    user_triplets = []
+    for entity in entity_values:
+        for rel, id in relationship_entity_id_pair:
+            if entity["@id"] == id:
+                user_triplets.extend([("User", rel, entity.get("substr", entity.get("Name")))])
+
+    return user_triplets
+
+
+def convert_triplets_to_natural_language(triplets: List[tuple]) -> List[str]:
+    contexts = [
+        [],
+    ]
+    prompts = [
+        # "Generate natural language sentences based on the following triplets. One sentence for each triplets"
+        "Translate each semantic triple into a sentence. Triplets"
+        f" : {triplets}"
+    ]
+    try:
+        response = requests.post(
+            GENERATIVE_SERVICE_URL,
+            json={
+                "dialog_contexts": contexts,
+                "prompts": prompts,
+                "configs": [GENERATIVE_SERVICE_CONFIG] * len(contexts),
+                "openai_api_keys": envvars_to_send * len(contexts),
```

Comment:
this one will not work if you replace openai with anthropic, for example.
https://github.com/deeppavlov/dream/blob/dev/common/prompts.py#L54
and 
https://github.com/deeppavlov/dream/blob/dev/common/prompts.py#L14
find an example of usage anywhere in the dream by search

Reply:
Done